airflow/operators/python.py

```diff
@@ -58,6 +56,15 @@
 from airflow.utils.process_utils import execute_in_subprocess
 from airflow.utils.python_virtualenv import prepare_virtualenv, write_python_script
 
+log = logging.getLogger(__name__)
+
+if shutil.which("cloudpickle") or importlib.util.find_spec("cloudpickle"):
+    import cloudpickle as serialization_library
+elif shutil.which("dill") or importlib.util.find_spec("dill"):
+    import dill as serialization_library
+else:
+    log.warning("Neither dill and cloudpickle are installed. Please install one with: pip install [name]")
```

Comment:
I am not sure if we should raise warning here. It will be always raised, even if you do not want touse dill/cloudpickle. Switching to DEBUG is likely better approach here.

Reply:
Fixed that. Thank you!